lightkurve/periodogram.py

```diff
@@ -725,6 +725,10 @@ def from_lightcurve(lc, minimum_frequency=None, maximum_frequency=None,
         """
         # Input validation
         normalization = validate_method(normalization, ['psd', 'amplitude'])
+        if (~np.isfinite(lc.flux)).any():
```

Comment:
`remove_nans` will only remove nans, not infinite values, so I think we should do this?
```suggestion
        if np.isnan(lc.flux).any():
```